Analysis/Tasks/PWGHF/HFD0CandidateSelector.cxx

```diff
@@ -324,11 +324,17 @@ struct HFD0CandidateSelector {
 
     for (auto& hfCandProng2 : hfCandProng2s) { //looping over 2 prong candidates
 
+      statusD0 = 0;
+      statusD0bar = 0;
+
+      if (!(hfCandProng2.hfflag() & 1 << D0ToPiK)) {
```

Comment:
What is the operation priority here?
Is this equivalent to `if (!(hfCandProng2.hfflag() & (1 << D0ToPiK)))` or to `if (!((hfCandProng2.hfflag() & 1) << D0ToPiK))`?

Reply:
it's the first one. I left shift 1 by my nbit (in this case D0ToPiK)  first.

It is equivalent to what we have now but using the enumerator for the bit.
https://github.com/AliceO2Group/AliceO2/blob/dev/Analysis/Tasks/PWGHF/HFLcCandidateSelector.cxx#L292

Reply:
So it looks like the shift `<<` has precedence over the AND `&`.
https://en.cppreference.com/w/cpp/language/operator_precedence
So it should be fine. :-)